lib/util/config_utils.js

```diff
@@ -101,6 +101,10 @@ shaka.util.ConfigUtils = class {
         shaka.log.alwaysWarn(
             'Unexpected number of arguments for ' + subPath);
         destination[k] = source[k];
+      } else if (Array.isArray(destination[k])) {
+        // Make a copy of the input array, so that changes to the source object
+        // don't immediately affect the running config.
+        destination[k] = source[k].slice();
```

Comment:
Do the array items need to be cloned as well?

Reply:
Right now, they are all arrays of strings.